pkg/node/node.go

```diff
@@ -280,12 +280,13 @@ func NewBee(o Options) (*Bee, error) {
 		return nil, fmt.Errorf("pullsync protocol: %w", err)
 	}
 
-	puller := puller.New(puller.Options{
-		StateStore: stateStore,
-		Topology:   topologyDriver,
-		PullSync:   pullSync,
-		Logger:     logger,
-	})
+	pullerOpts := puller.NewOptions()
+	pullerOpts.StateStore = stateStore
+	pullerOpts.Topology = topologyDriver
+	pullerOpts.PullSync = pullSync
+	pullerOpts.Logger = logger
```

Comment:
what is the criterion that determines that `Options.Bins` and `Options.ShallowBinPeers` should have default values in the options constructor, but not say, for example, `Options.Topology`?

Reply:
i now see @janos has made a somewhat similar comment